File: api-docs/packages/foodlk-api-docs/python/smoke.py
"""Quick client smoke — polite delay on."""

from __future__ import annotations

import json

from foodlk_api_docs import FoodlkApiDocsClient


def json_preview(data: object, limit: int = 200) -> str:
    try:
        return json.dumps(data, ensure_ascii=False)[:limit]
    except TypeError:
        return str(data)[:limit]


with FoodlkApiDocsClient(default_delay_seconds=1.0) as client:
    print("smoke", client.slug, "->", "openapi")
    data = client.openapi()
    preview = data[:200] if isinstance(data, str) else json_preview(data)
    print("ok", preview)

File: api-docs/packages/leco-outages-docs/javascript/examples/smoke.mjs
import { LecoOutagesDocsClient } from "../client.mjs";

const client = new LecoOutagesDocsClient({ defaultDelayMs: 1000 });
console.log("smoke", LecoOutagesDocsClient.slug, "->", "interruptionNotices");
const data = await client.interruptionNotices();
const preview = typeof data === "string" ? data.slice(0, 200) : JSON.stringify(data)?.slice(0, 200);
console.log("ok", preview);
